src/components/Animation/Animation.tsx

```diff
@@ -1,16 +1,26 @@
 import * as PropTypes from 'prop-types'
 import * as React from 'react'
 
-import { UIComponent, customPropTypes, childrenExist } from '../../lib'
+import {
+  UIComponent,
+  customPropTypes,
+  childrenExist,
+  StyledComponentProps,
+  commonPropTypes,
+} from '../../lib'
 import { AnimationProp } from '../../themes/types'
 import createAnimationStyles from '../../lib/createAnimationStyles'
-import { ChildrenComponentProps, StyledComponentProps } from '../../lib/commonPropInterfaces'
-import { styledComponentPropTypes, childrenComponentPropTypes } from '../../lib/commonPropTypes'
 
-export interface AnimationProps extends StyledComponentProps<any, any>, ChildrenComponentProps {
+export interface AnimationProps extends StyledComponentProps {
   /** An element type to render as (string or function). */
   as?: any
 
+  /**
+   *  Content for childrenApi
+   *  @docSiteIgnore
+   */
+  children?: React.ReactChild
```

Comment:
as description is duplicated now, I would rather suggest to still encapsulate it at the single place (`ChildrenComponentProps`), and just introduce optional generic parameter to this type:

```ts
ChildrenComponentProps<TChildren=React.ReactChildren> {
  /**
   *  Content for childrenApi
   *  @docSiteIgnore
   */
  children?: TChildren
}
```